startup_runners/outer_startup_runner.c

```diff
@@ -35,7 +35,7 @@ main(int argc, char **argv)
 {
     int result, num_per_core_data;
 
-    if (argc != 2) {
+    if (argc < 2) {  // krun passs more args, which are ignored here.
```

Comment:
The comment still doesn't make sense. If Krun passes (say) 5 arguments, the program will raise an error. Something doesn't add up to me.